templates/run-glue-crawler.js.tpl

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const AWS = require('aws-sdk')
+// Set the region
+AWS.config.update({region: process.env.AWS_REGION})
+
+const glue = new AWS.Glue()
+const crawlerName = '${glue_crawler_name}'
+
+exports.handler = async (event, context) => {
+  const { body } = event.Records[0]
+  const parsed_body = JSON.parse(body)
+
+  if (parsed_body.table !== 'signatures' || parsed_body.kind !== 'full') {
+    const logMessage = `Ignoring notification for table $${parsed_body.table} and kind $${parsed_body.kind}`
```

Comment:
won't this print `$` signs in the logs?

Reply:
No, this is for escaping the terraform template, otherwise terraform fails to render the template as the `parsed_body.table` variable is not provided. I verified output script is correct.

Reply:
aha!